### DISCUSSION TITLE
[Idea] Inventory Crontasks and Schedule Tasks

### DISCUSSION THREAD

**Discussion by FredVRA** [glpi-project/glpi-agent#387] [Ideas]
### Your idea

Hi,
Hi,
Last year, we had requested a development in Fusion inventory plugin in order to report scheduled tasks and server crontasks. Would it be possible to add it to the GLPI agent and the native GLPI inventory?

Name | Comments | ENABLED | EXECUTION USER | COMMAND | MINUTE | HOUR | DAY | MONTH | WEEK DAY | YEAR | STORAGE | STORAGE USER | Computer
Thanks,

**Comment by g-bougard** [comment]
Hi @FredVRA 
this was a pure FI development. And there's actually no such object support in GLPI.
On the agent side, the development made is still in a PR set with `[wip]` in the title and the developer totally ignored all my comments on his PR... Check it there: fusioninventory/fusioninventory-agent#900

Actually, there's no plan for such feature integration in GLPI10 & GLPI-Agent. Please contact our professional services if your company wants to invest in that feature.

**Reply by FredVRA** [reply]
Bonjour @g-bougard ,

Merci de ta réponse, dommage, il s'agit du seul blocage de notre côté pour passer à l'inventaire natif GLPI et depuis la mise à jour vers la version GLPI 10.0.7, nos inventaires ne remontent plus avec le plugin Fusion Inventory. 

En ce qui concerne la commande d'une prestation, nous l'avons déjà payé pour la développer sur Fusion inventory, cela signifie payer 2 fois pour la même fonctionnalité, c'est un peu abusé...

Cordialement,

**Reply by g-bougard** [reply]
Salut,
je comprends très bien ta frustration. Mais ce ne serait pas payer 2 fois la même fonctionnalité, mais payer l'adaptation d'une solution qui n'est plus maintenue pour une intégration pérène dans GLPI. On ne partirait pas de zéro puisque la phase de spécification a été faite et une implémentation existe qu'on devrait arriver à porter sans trop de problème et trop d'effort. Là, on retrouve bien tout l'intérêt d'avoir commandé du code sous licence libre.
Cordialement

**Reply by cedric-anne** [reply]
Le plugin `glpi-inventory` a commencé à bifurquer du plugin `fusioninventory-for-glpi` un peu avant la version 9.5+4.0 (à partir du commit [d18957dff37f25591c62b43f4c3aac45a7c3ec8e](https://github.com/fusioninventory/fusioninventory-for-glpi/tree/02fc6237b72cb06c5f8030382f8833516e0ff84b), datant du 21/06/2021).

Tout ce qui a été fait ensuite dans le plugin `fusioninventory-for-glpi` n'a pas été repris, pour diverses raisons, mais principalement parce que nous n'avons pas forcément suivi ce qui y a été fait.

Côté agent, je n'ai pas les détails, mais c'est probablement une situation similaire.

Si vous avez payé pour ce développement et que vous souhaitez le voir arriver dans GLPI Agent et le coeur de GLPI, le mieux est probablement que vous contactiez votre prestataire pour voir quelles solutions s'offrent à vous.
Le code de GLPI Agent, de GLPI et du plugin `glpi-inventory` est open source et nous sommes toujours heureux de recevoir des contribution de tiers, dès lors que celles-ci respectent nos standards de qualité.

**Comment by FredVRA** [comment]
Bonjour, c'est compris, en tout cas, merci pour vos réponses. 
Bonne journée,
